doubleml/double_ml.py

```diff
@@ -434,7 +456,7 @@ def __psi_deriv(self):
     def __all_se(self):
         return self._all_se[self._i_treat, self._i_rep]
 
-    def fit(self, n_jobs_cv=None, store_predictions=False, store_models=False):
+    def fit(self, n_jobs_cv=None, store_predictions=True, store_models=False):
```

Comment:
Thanks @SvenKlaassen  - I think it's a good idea to set the default for `store_predictions` to `True`  👍